### DISCUSSION TITLE
DISTANCES LUES SUR F-PLN ENTRE WPT FAUSSES

### DISCUSSION THREAD

**Discussion by kampinod** [flybywiresim/aircraft#4497] [Q&A] [answered]
Sur le F-PLN j'ai constaté plusieurs fois que les distances entre les WPT qui précédent l'APPROCHE sont nettement surestimées, ce qui complique beaucoup le calcul du TOD...
Exemples : 
LFRN-LFRB- Distance ARE/RB403= 74 au lieu de 6 Nm???
LFRD-LFRN. Approche RN 404(IAF) / IRN28(IF) pour la RWY28, distance affichée 21 (avant de passer le WPT précédent), distance réelle 6Nm (affichée au passage du WPT précédent)
LFBO et LFML, RIKSI>AVN passe de 82 à 9 Nm à l'approche de RIKSI, la distance restante est bloquée à 122 Nm et passe de 122 à 49 NM
Il y a d'autre exemples que je pourrais citer...
Merci d'avance pour votre réponse...

**Comment by CptBayer** [comment]
in english please :-p ...

Greets Klaus

**Reply by ExampleWasTaken** [reply]
just put it into google translate

**Reply by CptBayer** [reply]
trop épuisant :-p ...

**Comment by ExampleWasTaken** [comment, marked as answer]
Comme je ne parle pas français, je suggère de traduire avec Google Traduction.

The current flight plan manager is still the default of the default A320 and is full of bugs. A new, custom one is currently in development.

**Comment by kampinod** [comment]
Thank you very much for your response, I understood that there are still a lot of bugs on the A320 flight plan ....
I understand that it is not easy, but I hope that this bug will be solved soon because it is penalizing for the calculation of the TOD, essential element of a flight in A320 ...
Thank you again for all the work done by the community.

**Reply by ExampleWasTaken** [reply]
The new custom flight plan manager is scheduled to come to the experimental version very soon! So there is a light on the horizon ;)

**Reply by kampinod** [reply]
Thank you very much for your response et your help.
